lib/MusicBrainz/Server/Form/Role/ValidateUsername.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Form::Role::ValidateUsername;
```

Comment:
I don’t think this is appropriate to have as a role. Make it a helper function somewhere.